### PR TITLE
808 retrieve multiple scores

### PR DESCRIPTION
**Pull request by schultztimothy**

### CODE REVIEW

File: api/registry/api.py [hunk 1]
```diff
@@ -243,3 +262,50 @@ def get_score(request, address: str, community_id: int) -> ScoreResponse:
...
+
+
+class ScoreFilters(Schema):
+    community_id: int
+    limit: int = 50
+    offset: int = 0
+    addresses: List[str] = Field(None, alias="addresses")
+
+
+@router.get("/scores", auth=ApiKey(), response=List[ScoreResponse])
```
Comment (nutrina) [inline api/registry/api.py:274]: Let us use the path `/score` not `/scores` (I think this is one of the general naming conventions).

File: api/registry/api.py [hunk 2]
```diff
@@ -243,3 +262,50 @@ def get_score(request, address: str, community_id: int) -> ScoreResponse:
...
+def get_scores(request, filters: ScoreFilters = Query({})) -> List[ScoreResponse]:
```
Comment (nutrina) [inline api/registry/api.py:275]: In my opinion this should be a read function that returns a list of items, and allows filtering, and we should keep it somewhat generic.
So:
1.  the filter attributes addresses and community_id should be optional
2. if no filter is specified, it is ok to return the entire list of results
3. the current implementation does not restrict. I don't think this is an explicit requirement, but given that we use the api key, we should use this restriction.

**Review by nutrina — COMMENTED** [review]
I have added some comments

**Comment by schultztimothy** [comment]
All of the comments should be accounted for

**Review by nutrina — APPROVED** [review]
Looks good.

**Pull request merged**